pkg/scanners/scannerv4/nodescan_convert.go

```diff
@@ -113,3 +121,20 @@ func toStorageNotes(notes []v4.VulnerabilityReport_Note) []storage.NodeScan_Note
 	}
 	return convertedNotes
 }
+
+func fixNotes(notes []storage.NodeScan_Note, osImageRef string) []storage.NodeScan_Note {
+	isRHCOS := strings.HasPrefix(osImageRef, "Red Hat Enterprise Linux CoreOS")
+	if !isRHCOS {
+		// Don't change notes if scanned node isn't RHCOS
+		return notes
+	}
```

Comment:
Let's avoid `don't` and `isn't` in one sentence. Optionally, we could also get rid of `isRHCOS` var if not used later. 
 ```suggestion
	if !strings.HasPrefix(osImageRef, "Red Hat Enterprise Linux CoreOS") {
		// Keep the notes for nodes other than RHCOS
		return notes
	}
```